README.md

```diff
@@ -1,35 +1,36 @@
 # Bitcoin Design - Meta
 `community`, `project-management`, `process`
 
-Anything related to group consensus and coordination of the Bitcoin Design Community.
+Activity in the Bitcoin Design community largely happens in the Slack workspace and here on Github, but also in the various projects our members are involved in.
 
-### Getting Started
+This repository is used for organizing calls, group consensus and coordination. The [Guide](https://github.com/BitcoinDesign/Guide) repository is for the Bitcoin Design Guide we are working on, which is a design reference for anyone building open-source bitcoin products.
 
-#### How to get involved in the community
+Community activity is fluid, but here are a few important starting points:
 
-1. [Join the community](https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA) on Slack
-2. Learn a pr [Project Life Cycle]((Projects.md))
-3. Discuss and shape [processes for the community and open design](https://github.com/BitcoinDesign/Meta/issues)
+## How to get involved in the community
 
-#### How to get involved in projects
+1. [Join the community on Slack](https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA) on Slack
```

Comment:
remove the redundant "on Slack" after the link

Reply:
Thanks. Fixed.